docs/cookbook/platform-sh-advanced.rst

```diff
@@ -0,0 +1,194 @@
+Advanced Platform.sh configurations
+===================================
+
+The basic set-up let's you easily set-up a Platform.sh project running your Sylius project. It should give you an
+environment suitable for testing Platform.sh in combination with Sylius.
+
+In this guide additional tips will be given in order to benefit in a production environment.
+
+Keep sessions between deployments
+---------------------------------
+
+The default configuration saves PHP sessions into ``/tmp/sessions``. Platform.sh functions in such way that each
+deployment spins up a new container instance and therefore the temporary folder holding sessions will be gone.
+
+In order to save the PHP sessions on disk, the following steps need to be followed:
+
+* In ``platform.app.yml`` add the following under the mount property:
+
+.. code-block:: yaml
+
+    mount:
+        "/app/sessions": "shared:files/sessions"
+
+
+* In the ``app/config/parameters_platform.php`` replace the session path:
+
+.. code-block:: php
+
+    ini_set('session.save_path', '/app/app/sessions');
+
+Alternatively you can use a ``php.ini``` file in the root of your project:
+
+.. code-block:: ini
+
+    session.save_path = "/app/app/sessions"
+
+Let Platform.sh generate your application secret:
+-------------------------------------------------
+
+The application secret is used in several places in Sylius and Symfony. Platform.sh allows you to deploy an
+environment for each branch you have, and therefore it makes sense to have a secret automatically generated by the
+Platform.sh system.
+
+Add the following lines to ``app/config/parameters_platform.php`` to make use of the `variables Platform.sh offers <https://docs.platform.sh/development/variables.html#platformsh-provided-variables>`_:
+
+.. code-block:: php
+
+    if (getenv('PLATFORM_PROJECT_ENTROPY')) {
+        $container->setParameter('secret', getenv('PLATFORM_PROJECT_ENTROPY');
+    }
+
+Use Redis for Doctrine caching:
+-------------------------------
+
+Want to use the metacache, query cache or result cache Symfony and Doctrine have to offer? It comes with a caveat.
+Platform.sh doesn't allow you to connect to all your services yet from inside the `build` hook. The following
+tutorial will guide you through this and make use of Redis. In the default example Redis is already activated.
+
+* On top of your ``app/config/parameters_platform.php`` add:
+
+.. code-block:: php
+
+    $container->setParameter('metacache_driver', 'array');
```

Comment:
Won't it be better to have this values as default parameters in parameters.yml.dist file? Also, it should allow building your app outside platform.sh

Reply:
Haven't thought about that. I think it's safe to move them in there, isn't it? I don't see downsides atleast..